config/manifests/xdescriptors_patch.yaml

```diff
@@ -1,6 +1,9 @@
-- op: replace
-  path: /spec/customresourcedefinitions/owned/2/specDescriptors/3/path
-  value: flightRecorder.name
+- op: add
+  path: /spec/customresourcedefinitions/owned/2/specDescriptors/3
+  value:
+    description: Flight Recorder Name
```

Comment:
For the description, maybe we could split the existing description for the `flightRecorder` field into two:
https://github.com/cryostatio/cryostat-operator/blob/7bb319658635c3f220da3d902b09880122f91fba/api/v1beta1/recording_types.go#L67-L68
Remove the second sentence from that file, and use it here. "Select the FlightRecorder with the name of the target Pod for this Recording."

Reply:
@ebaron So the patch would become as below?

```yaml
value:
    description: Select the FlightRecorder with the name of the target Pod for this Recording.
    displayName: Name
    path: flightRecorder.name
    x-descriptors: [ "urn:alm:descriptor:io.kubernetes:operator.cryostat.io:v1beta1:FlightRecorder" ]
```

Reply:
Right